packages/bpk-component-carousel/carousel.scss

```diff
@@ -0,0 +1,36 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '~bpk-mixins/index';
+
+/* stylelint-disable unit-blacklist */
+.bpk-carousel-story__item {
+  width: 333px;
```

Comment:
We should use Backpack tokens here e.g. `$bpk-spacing-lg * 10`

Reply:
If this file is only used in stories we should follow convention and call it `stories.scss`.

Reply:
We don't have tokens that would make this value hence the hardcoded here so it follows the example designs - plus some Backpack components have values in px hardcoded too as the actual running component so not sure if as an example component this would be ok